pkg/reconciler/instances/istio/bootstrap_test.go

```diff
@@ -9,23 +11,23 @@ import (
 )
 
 func TestParsePaths(t *testing.T) {
-	alwaysValidFn := func(path string) error { return nil }
+	alwaysValidFn := func(path string, logger *zap.SugaredLogger) error { return nil }
```

Comment:
If we extract valid check out of `parsePaths`, we don't need this `alwaysValidFn`